nano/node/node.cpp

```diff
@@ -3534,6 +3534,28 @@ bool nano::active_transactions::active (nano::block const & block_a)
 	return roots.find (nano::uint512_union (block_a.previous (), block_a.root ())) != roots.end ();
 }
 
+uint64_t nano::active_transactions::active_difficulty ()
+{
+	nano::network_params params;
+	uint64_t result (params.publish_threshold);
```

Comment:
This can use `node.network_params.publish_threshold` instead of a new params instance.